server/api/products.post.ts

```diff
@@ -1,9 +1,40 @@
-import { PrismaClient } from '@prisma/client'
+import { PrismaClient, Product } from '@prisma/client';
 
 const prisma = new PrismaClient()
 
 export default defineEventHandler(async (event) => {
   const body = await readBody(event)
 
-  // Implement your API here
+  if (!body.title || !body.price || !body.description) {
```

Comment:
We could do 3 separate if statements, which I dont like because it repeats code and is hard to maintain if we would to alter Product attributes.
We could use validation library, like Zod, which is made for exactly this kind of checking. 
Or creating an array of required attributes and use that for checking the body, something like this:

const requiredFields = ['title', 'price', 'description'];
const missingFields = requiredFields.filter(field => !body[field]);

if (missingFields.length > 0) {
  const errorMessage = `Missing ${missingFields.join(', ')}`;
  return {
    statusCode: 400,
    body: errorMessage
  };
}